### PR TITLE
krakenx: init at 0.0.1

### PR DESCRIPTION
**Pull request by WilliButz**
This adds the python script `colctl` to control the colors as well as fan- and pump-speed of the `NZXT Kraken X62/X52` cpu coolers.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).
- [x] Tested color, fan- and pump-speed control functionality
---

### CODE REVIEW

File: pkgs/tools/system/krakenx/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{ lib, python3, python3Packages }:
```
Comment (nlewo) [inline pkgs/tools/system/krakenx/default.nix]: You can remove `python3` since it is not used

**Review by nlewo — CHANGES_REQUESTED** [review]

File: pkgs/tools/system/krakenx/default.nix (hunk 1 above)
Comment (WilliButz) [reply pkgs/tools/system/krakenx/default.nix]: Done. I was using `python3.pkgs` at first and forgot to remove it there %)

**Comment by WilliButz** [comment]
@nlewo ping :)

**Comment by nlewo** [comment]
@WilliButz arf, sorry I missed the notification :/
@GrahamcOfBorg build krakenx

**Comment by WilliButz** [comment]
@nlewo No problem :)

**Pull request merged**